process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+
+Projects follow the Sandbox, Incubating, Graduated, and Archived lifecycle stages defined below. Projects that seek widespread adoption and end user use are expected to progress through the stages. Projects coming to OpenSSF that already meet the entry requirements may enter the Incubating stage directly.
+
+<!-- TOC -->
+
+  - [Stages - Definitions & Expectations](#stages---definitions--expectations)
+      - [Sandbox](#sandbox)
+      - [Incubating](#incubating)
+      - [Graduated](#graduated)
+      - [Archived](#archived)
+
+<!-- /TOC -->
+
+
+## Stages - Definitions & Expectations
+
+A project's lifecycle stage is a signal of maturity level. Projects progress through the lifecycle by demonstrating their maturity in technical vision, impact to the open source community (such as adoption), and sustainability in governance and contributors. All projects, at all stages, are expected to abide by the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/).
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage Projects and has four goals:
+
+* Encourage experimentation and open collaboration that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project
+* Facilitate alignment with existing Working Groups or Projects if this is desired (there may be cases where pursuit of different approaches is desirable)
+* Nurture Projects
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct, and IP Policy requirements.
+
+#### Project Responsibilities
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project)
+* Provides project updates to OpenSSF Marketing Committee as requested
+
+#### Project Support
+* Receives guidance on technical direction from TAC
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Reserved space for project updates in OpenSSF newsletters
+* May request infrastructure support from the OpenSSF
+* Projects, sponsors, and contributors may not promote the project with the OpenSSF logo or be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." Projects may say they are, "A sandbox project in the OpenSSF" or "An experimental project in the OpenSSF."
+
+#### Sandbox Entry Requirements and Considerations
+
+* Projects must have a minimum of two maintainers with different company affiliations.
```

Comment:
Is two maintainers a reasonable requirement for sandbox? If a company wants to contribute, it seems like that would be a first step towards getting another maintainer. I think we should drop that *here*, and start ramping that up for the next stage. Otherwise projects may have trouble getting started.